libs/core/src/lib/list/list-item/list-item.component.ts

```diff
@@ -68,6 +68,10 @@ export class ListItemComponent implements KeyboardSupportItemInterface, AfterCon
     @HostBinding('class.fd-list__item--link')
     link = false;
 
+    /** Defines if list item is used inside MultiInputComponent */
```

Comment:
At the very least this should be hidden from the app developer, but I think we do not need to use an input at all, the multi-input could get list item content children and set the property there.  Or this list item could ascend a few parents and figure out if it is in a multi input.

Reply:
@mikerodonnell89 You are right, it should be controlled from parent. Anyway it has been changed, so now it uses selection list for multi input with some minor styles adjustments.